packages/edit-site/src/components/sidebar/global-styles/preview.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	__experimentalHStack as HStack,
+	__experimentalVStack as VStack,
+	Card,
+	ColorIndicator,
+} from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */ import { useGlobalStylesContext } from '../../editor/global-styles-provider';
+
+export const StylePreview = () => {
+	const { getStyle } = useGlobalStylesContext();
+	const fontFamily = getStyle( 'root', 'fontFamily' ) ?? 'serif';
+	const fontSize = getStyle( 'root', 'fontSize' ) ?? 'initial';
+	const textColor = getStyle( 'root', 'color' ) ?? 'black';
+	const linkColor = getStyle( 'root', 'linkColor' ) ?? 'blue';
+	const backgroundColor = getStyle( 'root', 'backgroundColor' ) ?? 'white';
```

Comment:
The fallback values for all the properties here are the browser defaults. Note that they can defer for the actual fallback in the theme if it's using CSS to define these.